config/main.py

```diff
@@ -6594,6 +6630,40 @@ def sample_rate(ctx, ifname, rate):
             except ValueError as e:
                 ctx.fail("Invalid ConfigDB. Error: {}".format(e))
 
+#
+# 'sflow' command ('config sflow interface sample-direction  ...')
+#
+@interface.command('sample-direction')
+@click.argument('ifname', metavar='<interface_name>', required=True, type=str)
+@click.argument('direction', metavar='<sample_direction>', required=True, type=str)
+@click.pass_context
+def sample_direction(ctx, ifname, direction):
```

Comment:
sample_direction() --> interface_sample_direction()

Reply:
will rename to interface_sample_direction()